app/api/routers/breeze_buddy/analytics/handlers.py

```
@@ -0,0 +1,392 @@
+    }
+
+    return {
+        "type": "conversion",
+        "filters_applied": filters,
+        "results": conversion_data
+    }
+
+
+async def get_performance_analytics(
+    filters: Dict[str, Any],
+    options: Dict[str, Any],
```

Ruff (0.14.8)

Unused function argument: `options`

(ARG001)